datadog/dogshell/common.py

```diff
@@ -10,23 +10,33 @@
 
 def print_err(msg):
     if is_p3k():
-        print('ERROR: ' + msg + '\n', file=sys.stderr)
+        print(msg + '\n', file=sys.stderr)
     else:
         sys.stderr.write(msg + '\n')
 
 
 def report_errors(res):
     if 'errors' in res:
-        for e in res['errors']:
-            print_err('ERROR: ' + e)
+        errors = res['errors']
+        if isinstance(errors, str):
+            print_err('ERROR: ' + errors)
+        elif isinstance(errors, list):
+            print('ERROR:')
+            for error in errors:
+                print_err(error)
```

Comment:
make it `print_err("ERROR: " + error)` and remove the  print('ERROR:')`. Same for warnings